include/libopencm3/stm32/common/usb_common_v2.h

```diff
@@ -0,0 +1,105 @@
+/** @addtogroup usb_defines
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* THIS FILE SHOULD NOT BE INCLUDED DIRECTLY, BUT ONLY VIA USB.H
+ * The order of header inclusion is important. usart.h includes the device
+ * specific memorymap.h header before including this header file.*/
+
+/** @cond */
+#ifdef LIBOPENCM3_USB_H
+/** @endcond */
+#ifndef LIBOPENCM3_USB_COMMON_V2_H
+#define LIBOPENCM3_USB_COMMON_V2_H
+
+#include <libopencm3/stm32/common/usb_common_all.h>
+
+/*****************************************************************************/
+/* Module definitions                                                        */
+/*****************************************************************************/
+
+/*****************************************************************************/
+/* Register definitions                                                      */
+/*****************************************************************************/
+
+#define USB_LPMCSR		MMIO32(USB_DEV_FS_BASE + 0x54)
+#define USB_BDCR		MMIO32(USB_DEV_FS_BASE + 0x58)
+
+/*****************************************************************************/
+/* Register values                                                           */
+/*****************************************************************************/
+
+/* USB_CNTR Values ----------------------------------------------------------*/
+
+#define USB_CNTR_L1REQM		(1 << 7)
+#define USB_CNTR_L1RESUME	(1 << 5)
+
+/* USB_ISTR Values ----------------------------------------------------------*/
+
+#define USB_ISTR_L1REQ		(1 << 5)
```

Comment:
This is wrong, should be 1<<7 (current code has it defined as 0x80 which is correct)